### DISCUSSION TITLE
Achievements: Heart On Your Sleeve

### DISCUSSION THREAD

**Discussion by thaissevero** [community/community#41863] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

<h1> 💕  Hello everyone    </h1>

**Does anyone know how to get the achievements: Heart On Your Sleeve and Open Sourcerer, please? 
I searched a lot, but I didn't find it.**

*It's so beautiful, I really want one of them on my profile haha*
##

<img align='center' src="https://media.tenor.com/9R3FrJ-5odgAAAAC/pretty-please-please.gif" />

**Comment by mohamedzhioua** [comment]
@thaissevero An available non-official repository list all the achievement [here](https://github.com/drknzz/GitHub-Achievements).
But as mentioned by the[ original blog post made by Github introducing the achievement badges](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/), they said that they will never tell how many badges they are OR how to unlock them.
For that, you have to wait until somebody share it on his profile.

**Comment by cihatsolak** [comment]
Hi. 
To earn such achievements, you must achieve the requirements. There is currently no detailed information about these two newly arrived badges. However, as in the link below, it was stated that no information will be given about this subject.

Link: [Introducing Achievements: recognizing the many stages of a developer’s coding journey](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/)

Take your time for this and keep coding. I'm sure you'll have the badges.

**Reply by thaissevero** [reply]
Ooh, thank youuu!!

**Reply by marse11e** [reply]
a

**Reply by Sriansh-raj** [reply]
Here is the public repo on list of available **GitHub achievements** and how to **get them**

Repo: **https://github.com/Sriansh-raj/GitHub-Achievements**

You can star it for your reference 🌟

Happy GitHub :octocat:

❤️

**Reply by fallenreds** [reply]
Thx!

**Reply by ais04134** [reply]
감사합니다.

**Reply by Ngochiendev** [reply]
nice

**Reply by patrickryan01** [reply]
❤️ Nice.

**Reply by olehrab** [reply]
I click star. I want to receive Heart On Your Sleeve.

**Reply by SaraSSC** [reply]
Nice 🙂

**Reply by ashwin-maurya** [reply]
❤️ Nice.

**Reply by akshitjain2004** [reply]
❤️ Nice

**Reply by Endrigogustavo** [reply]
❤️

**Reply by Al-vallon** [reply]
❤️

**Reply by SuperZekes** [reply]
❤

**Reply by imitatehappiness** [reply]
❤

**Reply by kangichu** [reply]
❤

**Reply by NadavIs56** [reply]
❤️

**Reply by PranavVerma-droid** [reply]
❤️

**Reply by PranavVerma-droid** [reply]
❤️

**Reply by MatiasNicolasAcevedo** [reply]
❤️

**Reply by MuhammadArifinHakim** [reply]
❤️

**Reply by ing9990** [reply]
❤️

**Reply by jahongirdev** [reply]
❤️

**Reply by Sachinthana-Lokuyaddage** [reply]
❤️

**Reply by J1NX-82** [reply]
❤️

**Reply by wesleybertipaglia** [reply]
Great 😃

**Reply by hoty1337** [reply]
❤️

**Reply by LinThitHtwe** [reply]
❤️

**Reply by mishmanners** [reply, marked as answer]
Heart on your Sleeve refers to how many hearts you give. Give more hearts and you'll get higher achievements. ❤️ for comments, issues, and more.

**Reply by marius-test** [reply]
I have a heart to everyone in this thread!

**Reply by MaulidanFazri** [reply]
♥️

**Reply by sam-ple** [reply]
❤️

**Comment by getlost01** [comment]
The Heart On Your Sleeve and Open Sourcerer achievements are not specific to any particular platform or service, so it is not clear how to obtain them. It is possible that these achievements are part of a custom game or program, or that they are used as part of a branding or marketing campaign by a company or organization. Without more information about the context in which these achievements are used, it is not possible to provide guidance on how to obtain them.

**Reply by thaissevero** [reply]
Oh I didn't know. Thank you!!!

**Reply by kah3vich** [reply]
thx

**Comment by SaileshBK** [comment]
you need a hoodie with long sleeves, then pull those sleeves up in your arms and start coding. once you achieve that the badge is yours. LOL, just kidding.

**Reply by thaissevero** [reply]
omg this is veeery easy!! HAHAHAHAH 😹😹

**Reply by SaileshBK** [reply]
However, there are other badges very easy to get.

**Reply by ILostMyMedic** [reply]
well I'm excited! got my Github hoodie shipping in the coming week! 😹

**Reply by marse11e** [reply]
hello

**Reply by fdhliakbar** [reply]
Hello guys

**Reply by Wanilldestroyer** [reply]
haha, good one

**Comment by jfullstackdev** [comment]
I'm wondering, can you have a screenshot of one GitHub user that achieved that badge ? maybe it's not on GitHub

**Reply by lotus647** [reply]
<img width="263" alt="Screenshot 2023-07-14 092113" src="https://github.com/community/community/assets/115119611/aac68a63-b807-4de4-99ec-5aeb2fbc76e4">
P.S. Yes I photoshopped this

**Reply by Cfomodz** [reply]
❤️

**Comment by gabrielmrts** [comment]
I was also wondering but apparently there is no known way to unlock

**Comment by ghost** [comment]
i found guy official YouTube GitHub 
https://youtube.com/shorts/1UgqjSCGslg?feature=share

**Reply by cihatsolak** [reply]
What does this have to do with what you're saying?

**Reply by ghost** [reply]
He has a new achievement

**Reply by cihatsolak** [reply]
no it is not.

**Reply by ghost** [reply]
can you check the YouTube video

**Reply by cihatsolak** [reply]
I'm sorry mate. I haven't seen. However, when I entered the man's account, I could not see the badge. I saw the badge in the video.

**Reply by ghost** [reply]
How this guy get the new badge it's really strange

**Reply by cihatsolak** [reply]
I think the guy is a microsoft employee. Might have done this to test it.

**Reply by ghost** [reply]
is beta testing is available for anyone it's possible to get the achievement before the release 🤔

**Reply by zbkllz** [reply]
he's like i wanna show this beta boring features just to show my beautiful achievements and i won't tell you how i got them you need to subscribe

**Reply by mrepol742** [reply]
The badges looks awesome.

**Comment by Tynab** [comment]
To earn the Heart on Your Sleeve achievement on GitHub, you need to publicly share a project on GitHub that contains a valid license and a README file. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Create a new project or select an existing project that you want to share on GitHub. The project should include a valid open source license and a README file that provides information about the project.
3. Create a new repository on GitHub and upload your project files.
4. Add a description and any other relevant information about your project, such as installation instructions or usage examples, to the README file.
5. Choose an appropriate open source license for your project and include a LICENSE file in the repository. GitHub provides several options for open source licenses that you can choose from.
6. Make sure that the repository is set to "public" so that others can view and access your project.
7. After you have completed these steps, you should receive the Heart on Your Sleeve badge on your GitHub profile.
Note that GitHub has specific requirements for what constitutes a valid open source license, so be sure to review the license options and choose one that fits the needs of your project. Additionally, it's important to provide clear and comprehensive information about your project in the README file to help others understand how to use and contribute to your project. Good luck!

**Reply by ghost** [reply]
is it possible.

**Reply by ProCNGit** [reply]
This does not work as per the comment instructions, and the person whom commented does not have it either.

**Reply by igorskyflyer** [reply]
You basically described what we all do on a daily basis here and yet none of us has the badge.

**Reply by lotus647** [reply]
ikr

**Comment by ProCNGit** [comment]
I feel these achievements are pretty much pointless because they can be easily gamed. Most of them can be achieved by just registering a second account and off you go. They mean nothing and do not reflect the true contributions of an account holder. This is just a tactic by GitHub to try get people to engage more with their platform. See it for what it is.

**Comment by fasilofficial** [comment]
To get the Heart on Your Sleeve achievement on GitHub, you need to add a README.md file to your profile repository that includes a non-empty personalized message in the "bio" field of your profile. Here are the steps:

Create a new repository named after your GitHub username. For example, if your username is "johnsmith", your repository should be named "johnsmith".

Add a README.md file to the root of the repository.

Edit the README.md file to include a personalized message in the "bio" field of your profile. For example: "Hi, I'm John Smith. I'm a software developer based in San Francisco, and I love building cool things with code!"

Commit and push the changes to your repository.

Go to your GitHub profile page and refresh it. You should now see the Heart on Your Sleeve badge next to your profile picture.

To get the Open Sourcerer achievement on GitHub, you need to make a significant contribution to an open source project. Here are the steps:

Find an open source project that you're interested in contributing to. You can search for projects on GitHub or other code hosting platforms.

Fork the project and make the necessary changes to fix a bug or add a new feature. Make sure to follow the project's contribution guidelines and coding standards.

Submit a pull request to the original project. Explain your changes and why they're important.

Wait for the project maintainer to review and merge your changes. If your pull request is accepted and merged, you will receive the Open Sourcerer badge on your GitHub profile.

Note that the Open Sourcerer achievement is awarded based on the quality and significance of your contribution, so it may take some time and effort to achieve.

**Comment by marse11e** [comment]
hello

**Reply by Aliton666** [reply]
hello

**Reply by Revaz-Goguadze** [reply]
hi

**Comment by marse11e** [comment]
1

**Reply by Aliton666** [reply]
2

**Reply by lotus647** [reply]
3

**Reply by yagiziskirik** [reply]
4

**Reply by Revaz-Goguadze** [reply]
5

**Reply by sam-ple** [reply]
5

**Comment by marse11e** [comment]
2

**Reply by Aliton666** [reply]
2

**Reply by sam-ple** [reply]
2

**Comment by marse11e** [comment]
3

**Reply by Aliton666** [reply]
3

**Comment by marse11e** [comment]
4

**Reply by Aliton666** [reply]
4

**Comment by marse11e** [comment]
5

**Reply by Aliton666** [reply]
5

**Comment by marse11e** [comment]
6

**Reply by Aliton666** [reply]
6

**Comment by mrepol742** [comment]
7

**Comment by montasim** [comment]
Hi @everyone, I am trying to get the "Heart On Your Sleeve" badge. How do I get it?

**Reply by mrepol742** [reply]
There is no way to get it for now. It's currently on testing and may only be visible on github staff. Mean's if they earned it, only them could see it. 

For example, this one:
https://github.com/damovisa

He have `Heart on your sleeve` and `Open Sourcerer` 3x, but it was not visible to us.

**Reply by montasim** [reply]
♥

**Comment by tammv** [comment]
8

**Comment by YasinDehfuli** [comment]
The heart of sleeve and and open sourcer badge wasn't available for now.

if you want to get other github badges. this repo helps you : 
https://github.com/4xmen/Get-Github-Achievements

**Comment by opticaldrive** [comment]
https://github.com/orgs/community/discussions/43514
Maybe something here?

**Comment by opticaldrive** [comment]
Aha!
<!DOCTYPE html><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; caret-color: rgb(31, 35, 40); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;">These achievements are still in testing and will be obtainable soon:</p>

Name | Badge Image | Earnable? | Earned By | Difficulty
-- | -- | -- | -- | --
Heart On Your Sleeve |   | 🔜 (In testing) | User cans obtain this badge by reacting to something on GitHub with a ❤️ emoji | Very Easy
Open Sourcerer |   | 🔜 (In testing) | Can be obtained if a user had Pull Requests merged in more than 1 public repository | Medium

<h1 tabindex="-1" dir="auto" style="box-sizing: border-box; font-size: 2em; margin: 24px 0px 16px; font-weight: var(--base-text-weight-semibold, 600); line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid var(--borderColor-muted, var(--color-border-muted)); caret-color: rgb(31, 35, 40); color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-caps: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration: none;"><a id="user-content--obtainable-achievements" class="anchor" aria-hidden="true" href="#-obtainable-achievements" style="box-sizing: border-box; background-color: transparent; color: var(--fgColor-accent, var(--color-accent-fg)); text-decoration: none; float: left; padding-right: 4px; margin-left: -20px; line-height: 1;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z"></path></svg></a><g-emoji class="g-emoji" alias="1st_place_medal" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f947.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1.25em; font-weight: var(--base-text-weight-normal, 400); line-height: 1; vertical-align: -0.075em; font-style: normal !important;"></g-emoji></h1><br class="Apple-interchange-newline">

**Reply by opticaldrive** [reply]
That was from:
https://github.com/Thinkright20/Profile-Badges

**Reply by lotus647** [reply]
kewl

**Reply by montasim** [reply]
❤️

**Reply by lotus647** [reply]
its not published yet lmao. wait until like next month

**Reply by ImeshaDilshani** [reply]
❤️

**Comment by NomanGhiasy** [comment]
React to something on GitHub with a ❤️ emoji

**Reply by montasim** [reply]
❤️

**Comment by 000xs** [comment]
❤

**Comment by Jaxiie** [comment]
I think it doesn't work right now

**Reply by LeleDallas** [reply]
I think so too...

**Comment by fedorkanan** [comment]
Hey @thaissevero , the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges are not yet released to public GitHub users at the moment.

Please kindly wait for & stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information (based on my understanding) on the mechanics to achieve those two new badges.

Hope this clarifies & answers your question, take care thaissevero 😄!

**Comment by kauaneiras** [comment]
These achievements may not be available yet. I haven't found any repositories where anyone has won.

**Reply by SuperZekes** [reply]
When they are you can look at the blog

**Comment by mhmetglrq** [comment]
These achievements not available yet.

**Comment by rajiv2004-cloud** [comment]
These achievements not available still in developement.

**Comment by ImeshaDilshani** [comment]
❤️

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @thaissevero. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
